README.md

```diff
@@ -1 +1,49 @@
-# replace this
\ No newline at end of file
+# replace this
+
+
+## Getting Started
+___
+
+### TypeScript
+
+#### Installation
+```shell
+$ npm install cdk-extensions
+```
+
+#### Usage
+<i>Minimal deployable example creates the default logging strategy defined in AwsLoggingStack for ALB, CloudFront, CloudTrail, Flow Logs, and S3 access logs. For each service, an S3 bucket is created and a Glue crawler to analyze and categorize the data and store the associated metadata in the AWS Glue Data Catalog. Default named queries have been defined for each AWS service. For more details on this and the other available stacks and constructs, consult the respective READMEs.</i>
```

Comment:
Do we want this in italics?  Should be standardized across all other READMEs.

Reply:
I think probably no. In an earlier iteration it was a much shorter statement and I was treating as a caption for the examples. I've dropped that and cleaned it all up to be more presentable and less repetitive